proto/stats.proto

```diff
@@ -143,6 +159,11 @@ message GetTrendResponse {
   repeated ExecutionStat execution_stat = 3;
 
   bool has_invocation_stat_percentiles = 4;
+
+  // A summary of metrics from the period specified in the query.
+  Summary summary_current = 5;
+  // A summary of metrics from the previous period.
```

Comment:
maybe just add a little bit here about this period being equal in length to the one the user requested.

Reply:
Done.